app/webpacker/src/stylesheets/_rules_of_origin.scss

```diff
@@ -27,3 +27,10 @@
     white-space: nowrap;
   }
 }
+
+.js-enabled .govuk-tabs__panel {
+  margin-bottom: 0;
+  padding: 30px 20px;
+  border: 1px solid #b1b4b6;
+  border-top: 0;
+}
```

Comment:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files